lib/tasks_private/instance_types.rake

```diff
@@ -0,0 +1,81 @@
+namespace 'aws:extract' do
+  desc 'Get / renew instance_types'
+  task :instance_types => [:environment] do
```

Comment:
Do you need environment?  I think you could drop this, and then just require activesupport selectively for things like where you need `.gigabyes`